plugins/ms-vscode.node-debug/1.0.0/meta.yaml

```diff
@@ -0,0 +1,13 @@
+id: ms-vscode.node-debug
+version: 1.0.0
+type: VS Code extension
+name: Node Debug (legacy)
+title: Node.js debugging support (legacy)
+description: Node debug (legacy) is the debugger for Node.js versions < 8.0. Together with Node Debug forms the Node.js debugging experience. https://marketplace.visualstudio.com/items?itemName=ms-vscode.node-debug
```

Comment:
quick question: for now the extensions are shipped with eclipse/che-theia-endpoint-runtime which is a "nodejs 8" version.
So how can we use that debugger that targets nodejs < 8.0 ?

Reply:
I mean, as we're always using nodejs >= 8.0 shouldn't we have a single plug-in referencing the two extensions ?

Reply:
I am not sure If I understand you correctly. 

> shouldn't we have a single plug-in referencing the two extensions ?

I don't know if it is possible to use a single container for several plugins. 

> So how can we use that debugger that targets nodejs < 8.0 ?

No way. But user has to enable both plugins. Legacy debugger (node debug) contains functionality that is needed for another one (node debug2).

Reply:
I was thinking about UX
if I want to enable nodejs debugging, I would prefer to only enable one plug-in to my che workspace that will bring two extensions.
And if I select only one of the two it will fail ?


Also about memory if these two related plug-ins are deployed on two sidecars it's using memory

Reply:
Completely agree. It is not obvious that user has to enable both plugins. 

> I would prefer to only enable one plug-in to my che workspace that will bring two extensions.

Do you know how to achieve that? From my understanding we need to write a  plugin that would download both extensions and activate them, right?

Reply:
yes correct.
The broker is downloading plug-ins so I don't think it's an issue but it may be an improvement if not supported right now. BTW we can revisit later as well (like for GA)